smartsim/_core/control/previewrenderer.py

```diff
@@ -103,9 +128,11 @@ def render(
     :type output_format: _OutputFormatString
     """
 
-    verbosity_level = _check_verbosity_level(Verbosity(verbosity_level))
+    Verbosity(verbosity_level)
```

Comment:
Is this statement doing anything? I think we want the whole thing below?

`env.globals["Verbosity"] = Verbosity(verbosity_level)`

Reply:
or, potentially, just `env.globals["Verbosity"] = verbosity_level` since we are being passed the right type and have no need to construct a new object

Reply:
I think the intention was originally to do validation, but let's be sure it's required. I think receiving a `Verbosity` parameter should make it unnecessary.

Reply:
For whatever reason, this is the way I got it to throw a value error when a verbosity isnt valid, but I will dig to make sure its necessary